recipes/cdat_info/build.sh

```diff
@@ -0,0 +1,214 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+
+export GIT_DESCRIBE=`git describe --tags`
+
+cat > cdat_info.py.in << EOF
```

Comment:
I would rather if this was sent upstream, if this is an essential part of installing `uv-cdat`, or if you dropped it as a script in the `SRC_DIR` to make it easier to review/update.